src/test/kotlin/no/nav/syfo/testhelper/generator/AktivitetskravGenerator.kt

```diff
@@ -1,9 +1,25 @@
 package no.nav.syfo.testhelper.generator
 
+import no.nav.syfo.aktivitetskrav.AktivitetskravService
 import no.nav.syfo.aktivitetskrav.domain.*
 import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.testhelper.ExternalMockEnvironment
 import no.nav.syfo.testhelper.UserConstants
 import java.time.LocalDate
+import java.util.UUID
+
+fun AktivitetskravService.createAktivitetskravForTest(vararg aktivitetskrav: Aktivitetskrav) {
+    ExternalMockEnvironment.instance.database.connection.use { connection ->
+        aktivitetskrav.forEach {
+            createAktivitetskrav(
+                connection = connection,
+                aktivitetskrav = it,
+                referanseTilfelleBitUUID = UUID.randomUUID()
+            )
+        }
+        connection.commit()
+    }
+}
```

Comment:
Hmm, denne vil ikke være tilgjengelig i prod-koden fordi den ligger i `/test`-mappa i stedet for `/main`?

Reply:
Tror ikke det, men som Anders påpeker nedenfor så er det ikke sikkert den trengs lengre.